### ISSUE TITLE
Can't know which unnamed lists are blocking

### ISSUE THREAD

**Issue by edgrd** [gorhill/uBlock#556]
When pages are blocked by custom / external lists, uBlock does not show from which list comes the rule, if the list doesn't have a "! Title: list name"

Would be nice if we could see the characters after the last slash of unnamed lists URLs, like:

http://sysctl.org/cameleon/hosts.win‎

shows: .../hosts.win

or perhaps show the full URL

![example](https://cloud.githubusercontent.com/assets/11416158/9031227/095a4dd0-3986-11e5-9059-e37c2ab69d72.png)

**Comment by gorhill** [comment]
What version of uBlock Origin?

**Comment by edgrd** [comment]
1.0.0.0

**Comment by gorhill** [comment]
This was fixed in 1.0.0.0: <https://github.com/gorhill/uBlock/issues/192>.

**Comment by gorhill** [comment]
This has been fixed in upcoming 1.0.0.1: #536.

**Comment by gorhill** [comment]
Closing as duplicate of #536.
